extensions/replication/utils/SetupReplication.js

```diff
@@ -127,6 +127,7 @@ class SetupReplication extends BackbeatTask {
                 source.transport,
                 `${source.vault.host}:${source.vault.adminPort}`,
                 source.credentials, internalHttps),
+            // What is the proper way to use vault with the target?
             target: target.isExternal ? undefined : _setupIAMClient(
                 target.transport, `${destHost.host}:${destHost.port}`,
```

Comment:
I had to change the host and port to use the source vault host and admin port in order to get this to work locally. I am not sure if there is a better method, or maybe it is not supported? I was maybe using a non-typical setup of replicating to the same cloud server on different buckets instead of two different instances.